deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -546,10 +559,62 @@ def wait_statistics_file():
                            defined by "logcollector.state_interval".
     """
     for _ in range(LOG_COLLECTOR_GLOBAL_TIMEOUT):
-        if path.isfile(LOGCOLLECTOR_STATISTICS_FILE):
+        if os.path.isfile(LOGCOLLECTOR_STATISTICS_FILE):
             break
         else:
             sleep(1)
 
-    if not path.isfile(LOGCOLLECTOR_STATISTICS_FILE):
+    if not os.path.isfile(LOGCOLLECTOR_STATISTICS_FILE):
         raise FileNotFoundError
+
+
+def macos_logger_message(message):
+    """Create a unified logging system log using logger tool.
+
+    Args:
+        message (str): Logger event message.
+    """
+    os.system(f"logger {message}")
+
+
+def macos_os_log_message(type, subsystem, category, process_name="custom_log"):
```

Comment:
Be careful with the name of the functions. They don't represent what they're doing and also the name can be confusing.